site/templates/film/item.jet

```diff
@@ -92,10 +92,29 @@
             </div>
           {{end}}
           {{if len(film.GetSubtitles()) > 0 }}
-            <div class="meta-detail-subtitle">
-              <h2>{{i18n("meta_detail_subtitles")}}</h2>
-              <p>{{ film.GetSubtitles() }}</p>
-            </div>
+            {{subtitles:= ""}}
+            {{captions:= ""}}
+            {{range film.SubtitleTracks}}
+              {{if len(.) >= 3}}
```

Comment:
I would prefer a named variable instead of `.`

Reply:
Agree, but it is idiomatic [[1](https://github.com/CloudyKit/jet/blob/master/docs/syntax.md#delimiters), [2](https://github.com/CloudyKit/jet/blob/master/docs/syntax.md#range)].

Reply:
Yes it is idiotic.